code/modules/biomatter_manipulation/toxic_biomass.dm

```diff
@@ -6,7 +6,7 @@
 	if(istype(victim))
 		var/hazard_protection = victim.getarmor(null, "bio")
 		if(!hazard_protection)
-			victim.apply_damage(damage, TOX)
+			victim.apply_damage(damage * victim.reagent_permeability(), TOX)
```

Comment:
That's barely realated to PR but do you know what in theory `reagent_permeability` can return value higer than 1?